src/Testing/MakesGraphQLRequests.php

```diff
@@ -79,9 +79,9 @@ protected function postGraphQL(array $data, array $headers = [])
      * This is used for file uploads conforming to the specification:
      * https://github.com/jaydenseric/graphql-multipart-request-spec
      *
-     * @param  array<string, mixed>|array<int, array<string, mixed>> $operations
-     * @param  array<string, string> $map
-     * @param  array<int, \Illuminate\Http\Testing\File>  $files
+     * @param  array<string, mixed>|array<int, array<string, mixed>>  $operations
+     * @param  array<int, array>  $map
```

Comment:
```suggestion
     * @param  array<string, array<int, string>>  $map
```

Integer keys are erased by `json_encode`, which is why all examples use quoted numbers `'0'`. The values are indeed an array (to be precise: a list of strings), good catch.